src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -819,6 +821,14 @@ private TypedConstant GetDefaultValueArgument(ParameterSymbol parameter, Attribu
                 kind = TypedConstantKind.Primitive;
                 defaultValue = ((ContextualAttributeBinder)this).AttributedMember.GetMemberCallerName();
             }
+            else if (!IsEarlyAttributeBinder && syntax.ArgumentList is not null &&
+                getCallerArgumentArgumentIndex(parameter, argumentsToParams) is int argumentIndex && argumentIndex > -1 && argumentIndex < syntax.ArgumentList.Arguments.Count)
```

Comment:
>syntax.ArgumentList.Arguments.Count [](http://example.com/codeflow?start=139&length=35)

This looks suspicious. Isn't this going to include property/field initializers? Do we have tests for scenarios like that?

Reply:
> this going to include property/field initializers

Indeed, but it's probably that `getCallerArgumentArgumentIndex` won't return the index of a property/field initializer. But I'll give a deeper look and add more tests.

Reply:
@AlekseyTs Fixed. Thanks!